graphql-server/src/schemas/queries/requests.ts

```diff
@@ -0,0 +1,158 @@
+import {
+  DashboardRequest,
+  QueryDashboardRequestsArgs,
+} from "../../generated/graphql";
+import { neo4jDriver } from "../../utils/servers";
+import {
+  buildFinalCypherFilter,
+  getNeo4jCustomSort,
+  buildCypherDateFilter,
+  buildCypherBooleanFilter,
+} from "../custom";
+
+export function buildRequestsQueryBody({
```

Comment:
We previously had `buildRequestsQueryBody()` and `queryDashboardRequests()` as separate functions so we can reuse `buildRequestsQueryBody()` across (1) the subquery that fetches the request records and (2) the subquery that fetches the total number of requests. Now that we're getting rid of the 2nd subquery, I wonder if we should combine/refactor these functions.

This comment can be applied to the patients, cohorts, and samples query as well.

Reply:
I actually think it looks a little more organized this way